src/ex14_js-context/task-01.js

```diff
@@ -0,0 +1,67 @@
+var Calc = function(){
+    this.currentState = 0;
+
+    this.add = function(value) {
+        if (typeof value !== 'undefined') {
```

Comment:
let's add private method that checks if a value isn't undefined, and use it everywhere we check it

Reply:
better -- to check if the value is number because of calculator purpose